pkg/virt-controller/services/template.go

```diff
@@ -1813,11 +1814,15 @@ func getRequiredCapabilities(vmi *v1.VirtualMachineInstance, config *virtconfig.
 	}
 	// add a CAP_SYS_NICE capability to allow setting cpu affinity
 	capabilities = append(capabilities, CAP_SYS_NICE)
+	// add CAP_SYS_PTRACE capability needed by libvirt + swtpm
+	capabilities = append(capabilities, CAP_SYS_PTRACE)
 	// add CAP_SYS_ADMIN capability to allow virtiofs
 	if util.IsVMIVirtiofsEnabled(vmi) {
 		capabilities = append(capabilities, CAP_SYS_ADMIN)
 		capabilities = append(capabilities, getVirtiofsCapabilities()...)
 	}
+	if vmi.Spec.Domain.Devices.TPM != nil {
+	}
```

Comment:
Perhaps CAP_SYS_PTRACE should go under this `if...`

Reply:
Haha oops, gotta remove that block.
Unfortunately it can't, since the binary has the cap, the container needs it too, always...